content/tracing/advanced_usage/_index.md

```diff
@@ -3,6 +3,83 @@ title: Advanced Languages instrumentation
 kind: documentation
 ---
 
+## Trace Analytics Configuration
+
+Trace clients can send unsampled trace data to Trace Analytics for querying. Apply the following config to your application to enable this feature.
```

Comment:
@andrewsouthard1 I think in this overview we should explain a bit more what Trace Analytics does?

Reply:
Yeah, let's give a recap of what Trace Analytics is here.

>[Trace Analytics](https://docs.datadoghq.com/tracing/visualization/search/) is used to filter APM Data by [user-defined tags](https://docs.datadoghq.com/tracing/advanced_usage/?tab=java#custom-tagging) such as `customer_id`, `error_type` or `app_name` to help troubleshoot and filter your requests. Apply the following config to your application to enable this feature.

Reply:
Done.